kcidb/test_orm.py

```diff
@@ -498,3 +498,494 @@ def test_repr(self):
         self.assertEqual(repr(pattern(None, True, "checkout",
                               {("",)})),
                          ">checkout[\"\"]#")
+
+
+def raw_checkout(**kwargs):
+    """
+    Template for checkout type
```

Comment:
Please word the function description in imperative mood (here and everywhere else). I.e. like a command. E.g. this one could be "Generate raw data for a checkout from values for some fields". See also documentation for most other functions for reference.